modules/ROOT/pages/message-flows.adoc

```diff
@@ -0,0 +1,36 @@
+= Message Flows
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+To receive messages from a trading partner, create a message flow and configure the sections for the source, map, and target.
```

Comment:
Suggested refocus to accommodate a more useful title:

A _message flow_ is a communication vehicle between your organization (the _host_) and an enterprise or customer with whom your organization conducts trade (your _partner_). 

When you create a message flow, you configure three sections:

* Source
   Origin of the message flow, defined by the partner, host receiving endpoint (typically an API), and partner message type
* Map
   Transforms received message contents into a format you can use
* Target
   Destination of the message flow, defined by a host message type and endpoint (typically a host API)

Reply:
The next few lines, up to requirements, are copyedited only, in case you do not adopt suggested rewording.